scripts/test/partitionSplitting/system-test-instance.properties

```diff
@@ -65,162 +65,21 @@ sleeper.optional.stacks=SystemTestStack,CompactionStack,IngestStack,PartitionSpl
 # Whether to keep the sleeper table bucket, Dynamo tables related to tables and query results bucket or not.
 sleeper.retain.infra.after.destroy=false
 
-# The maximum number of concurrent compaction tasks to run
-sleeper.compaction.max.concurrent.tasks=300
-
 # Partitions with more than the following number of records in will be split
 # Set to 4 billion so that one round of splitting compactions happen
 sleeper.default.partition.splitting.threshold=20
 
 # How often partition splitting should happen
 sleeper.partition.splitting.period.minutes=10
```

Comment:
This won't be used as the system will be paused.

Reply:
Yes.